crawler/config_spec_and_defaults.conf

```diff
@@ -1,3 +1,15 @@
+[ general ]
+features_to_crawl = string_list(default=list('os', 'cpu'))
+environment = string(min=1, max=30, default='cloudsight')
+plugin_mode = boolean(default=False)
+
+plugin_places = string_list(default=list('plugins'))
+compress = boolean(default=False)
+
+link_container_log_files = boolean(default=False)
+default_mountpoint = string(default='Undefined')
+docker_containers_list = string(default='ALL')
```

Comment:
Yeah, it's just that couldn't think of another way of representing a list of all possible containers.